### ISSUE TITLE
Mongo API: replaceOne

### ISSUE THREAD

**Issue by ovu** [akka/alpakka#2100]
Hi all,

I am using the version 2.0.0-M2 and I realized that updateOne can update just one attribute in the collection in the Sink.

Here I have two questions:
1. Why updateOne does not support a list of attributes to be updated? 
2. I am missing is the replaceOne method that is available in the scala driver. Are you planning to implement it?

Thanks for your help!

**Comment by ovu** [comment]
I realized that actully the update of several attributes can be done in the following way:
        val updates = new BasicDBObject()
        updates.append("attr1", "New value")
        updates.append("attr2", "New value")
        updates.append("attr3", "New value"s)
        updates.append("attr4", "New value")
Sink.updateOne(filter, updates)

That was the answer to my first question. 
Regards

**Comment by seglo** [comment]
Hi @ovu.  I don't think there's a particular reason that a wrapper fir `replaceOne` has not been implemented.  Would you be interested in contributing it?

**Comment by jewertow** [comment]
@seglo I will implement this method.

**Comment by seglo** [comment]
Thanks @jewertow!

**Comment by jewertow** [comment]
@ovu
You can simply update multiple attributes using the builder method `Updates.combine`. For example:
```scala
Source(testRange).map(
        i =>
          DocumentUpdate(filter = Filters.eq("_id", i),
                         update = Updates.combine(Updates.set("attr1", "new value 1"), Updates.set("attr2", "new value 2")))
      )
```

**Comment by ennru** [comment]
Implemented with #2187
